index.ts

```
@@ -189,86 +198,71 @@
       config: {
-        uri: program.dbSource,
-        queries: program.dbQuery || "SELECT * FROM mytable",
+        uri: options.dbSource,
+        queries: options.dbQuery || "SELECT * FROM mytable",
       },
     },
   ];
 }
 
-const packageManager = !!program.useNpm
+const packageManager = !!options.useNpm
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)